### PR TITLE
Fix and test 2.0

### PR DESCRIPTION
**Pull request by eskaur**

### CODE REVIEW

File: scripts/sample_data.py [hunk 1]
```diff
@@ -39,6 +39,10 @@ def download_and_extract(file_camera_destination, point_cloud_destination):
         )
 
 
+def get_test_data_dir():
```
Comment (trym-b) [inline scripts/sample_data.py:42]:
```suggestion
def test_data_dir():
```

File: scripts/sample_data.py [hunk 2]
```diff
@@ -39,6 +39,10 @@ def download_and_extract(file_camera_destination, point_cloud_destination):
...
+    return (Path(__file__).parent / ".." / "test" / "test_data").resolve()
```
Comment (trym-b) [inline scripts/sample_data.py:43]:
```suggestion
    return (Path(__file__).parent.parent / "test" / "test_data").resolve()
```

File: test/conftest.py [hunk 3]
```diff
@@ -57,6 +57,47 @@ def frame_fixture(application, sample_point_cloud):  # pylint: disable=unused-ar
...
+    ]
+    return transforms
+
+
+@pytest.fixture(name="handeye_eth_transform")
+def handeye_eth_transform_fixture():
+    path = get_test_data_dir() / "handeye" / "eth" / "eth_transform.csv"
+    return np.loadtxt(str(path), delimiter=",")
+
+
```
Comment (trym-b) [inline test/conftest.py:100]: Nice

File: scripts/sample_data.py (hunk 2 above)
Comment (eskaur) [reply scripts/sample_data.py:43]: Yes of course. Thanks :)

**Comment by eskaur** [comment]
Resolved all your comments. Ready for next round, @trym-b .

**Review by trym-b — APPROVED** [review]
Looks good to me

**Pull request merged**